dashboard/app/reporting_email.go

```diff
@@ -1218,10 +1218,23 @@ func checkMailingListInCC(c context.Context, msg *email.Email, mailingList strin
 	return false
 }
 
-func warnMailingListInCC(c context.Context, msg *email.Email, mailingList string) string {
-	return fmt.Sprintf("Your commands are accepted, but please keep %v mailing list"+
-		" in CC next time. It serves as a history of what happened with each bug report."+
-		" Thank you.", mailingList)
+func forwardEmail(c context.Context, cfg *EmailConfig, msg *email.Email,
+	info *bugInfoResult, mailingList string) error {
+	log.Infof(c, "forwarding email: id=%q from=%q to=%q", msg.MessageID, msg.Author, mailingList)
+	body := fmt.Sprintf(`Forwarding an incoming command email to %v.
+
+***
+
+Subject: %s
+Author: %s
+
+%s`, mailingList, msg.Subject, msg.Author, msg.Body)
+	from, err := email.AddAddrContext(fromAddr(c), info.bugReporting.ID)
+	if err != nil {
+		return err
+	}
+	return sendMailText(c, cfg, msg.Subject, from,
+		[]string{mailingList}, info.bugReporting.ExtID, body)
```

Comment:
I think we also need to forward to LKML for linux reports.
1. We extract discussions from LKML.
2. Kernel devs may search LKML, and these will be searchable.
3. Lore should show them in the thread, that will be useful as well.

Reply:
I'm not sure we want to spam LKML that much? Any (especially older) LKML reports are never to be trusted to be still relevant anyway.

The purpose of the current code (before this PR) was to ask people to Cc us, so that at least it would be possible to look at the discussion at our googlegroup and see what has ever happened to the bug. I think we might want it even more for `#syz fix`, `#syz dup` and `#syz invalid` commands than for plain patch testing requests. This PR preserves this intention (without irritating syzbot replies).

Forwarding is also a potentially dangerous thing by itself (what if somebody deliberately sends rubbish to us?). By forwarding messages only to our group we limit the damage.

Reply:
LKML is just an archive, it's not list a list for reading each email. So sending emails there won't spam humans.

Reply:
By LKML you mean `linux-kernel@vger.kernel.org` only?

Reply:
I've updated the PR.

@dvyukov  PTAL